designer/server/src/common/components/pagination/template.njk

```diff
@@ -0,0 +1,27 @@
+{% from "govuk/components/pagination/macro.njk" import govukPagination %}
+
+{% if params.pagination and params.pagination.totalPages > 1 %}
+  {% set previous = null %}
+  {% if params.pagination.page > 1 %}
+    {% set previous = {
+      href: (params.baseUrl | default('')) ~ "?page=" ~ (params.pagination.page - 1) ~ "&perPage=" ~ params.pagination.perPage,
```

Comment:
What are these tildes, whitespace control? I can't see them documented in njk.

Reply:
For string concatenation. So for example: 
```
{% set firstName = "John" %}
{% set lastName = "Smith" %}

{# Using tilde for concatenation #}
{{ firstName ~ " " ~ lastName }}

{# Makes the ooutput: "John Smith" #}
```